content/workshops/openshift_service_mesh/lab1.3_deploymsa.md

```diff
@@ -58,10 +57,66 @@ spec:
 EOF
 ```
 
+<br>
+
+## Workshop Setup
+
+# <i class="fa fa-exclamation-triangle"></i> Only run this section if you are running this in an instructor-led workshop.
+# Skip this if you are running these labs on your own.
+
+<br>
+
+Set the project name variable.  If you are running this in an instructor-led workshop, the project name should match your username such as 'userX'.
+
+<blockquote>
+<i class="fa fa-terminal"></i>
+Set the project name variable.  For example:
+</blockquote>
+
+```
+PROJECT_NAME=userX
+```
+
+<br>
+
+Next we need a local copy of our application code.
+
+<blockquote>
+<i class="fa fa-terminal"></i> Clone the repository in your home directory:
+</blockquote>
+
+```
+cd $HOME
+git clone https://github.com/dudash/openshift-microservices.git
```

Comment:
they only need to clone in solo? Shouldn't this be required for both?

Reply:
If they're running it on their own, they clone the repo in the previous lab and install Istio using resources from the repo.  I'll add some steps to make sure they check they have the repo installed.

Reply:
OK, that makes sense. No need to add checks. Can you add the collapsable tags though?